### PR TITLE
update: permissions

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [ ] I checked the output.
- [ ] I applied the [style guide](styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
Preview: https://staceys-doc-1176.aiven-docs.pages.dev/docs/platform/concepts/permissions

**Review by rsalevsky — APPROVED** [review]

File: docs/platform/concepts/permissions.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* **Roles**: Sets of permissions that you can assign to a principal.
+
+Principals are
+[organization users](/docs/platform/howto/manage-org-users),
+[application users](/docs/platform/concepts/application-users),
+and [groups](/docs/platform/howto/list-groups).
+You can grant access to principals at the project level.
+
+To grant access to resources at the organization level, you can
+make organization users [super admin](/docs/platform/howto/make-super-admin).
```
Comment (harshini-rangaswamy) [inline docs/platform/concepts/permissions.md:18]:
```suggestion
assign [super admin](/docs/platform/howto/make-super-admin) role to organization users.
```
"make" can be ambiguous.

**Review by harshini-rangaswamy — APPROVED** [review]
Looks good! Just one small suggestion.

File: docs/platform/concepts/permissions.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/concepts/permissions.md:18]: It's the language we use in the UI and the doc that is linked `/docs/platform/howto/make-super-admin`.

**Pull request merged**